tests/test_bundled_label_operations.py

```diff
@@ -169,3 +169,28 @@ def test_bundled_label_save(save_label_rows_mock: MagicMock, project: Project):
     assert args is not None
     assert len(args["uids"]) == 3, "Expected 3 updates bundled"
     assert len(args["payload"]) == 3, "Expected 3 updates bundled"
+
+
+@patch.object(EncordClientProject, "save_label_rows")
+def test_bundled_label_save_with_explicit_bundle_size(save_label_rows_mock: MagicMock, project: Project):
+    label_rows = get_valid_label_rows(project)
```

Comment:
Might be worth asserting first on the length of the label_rows. Not clear just from reading what the length of label_rows is